dist/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Weather App</title>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div id="content">
+      <div class="container">
+        <div class="alert alert-success" style="display: none;" id="alert"></div>
+        <div class="alert alert-danger" style="display: none;" id="danger"></div>
+        <div class="row logo">
+          <div class="col-12 text-center">
+            <img src="https://www.wunderground.com/static/i/c/v4/28.svg" alt="logo" class="img-fluid">
+            <h1 class="display-1 font-weight-bold text-white">Weather App</h1>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-12 col-md-12 text-center search">
+            <form id="form">
+              <input type="text" name="search" value="" placeholder="Search for a city" id="searchInput"><br>
+              <button class="bg-blue pointer" type="submit" name="button" id="search">Search</button>
+            </form>
+            <p id="loading" class="text-white" style="display: none;">
```

Comment:
- [ ] Please remove the inline styling here. Consider moving it to the stylesheet.